examples/volumes/scaleio/README.md

```diff
@@ -0,0 +1,278 @@
+
+```
+$ kubectl create -f ./examples/volumes/scaleio/secret.yaml
+```
+
+## Deploying Pods with Persistent Volumes
+
+The following example shows how the ScaleIO volume plugin for Kubernetes automatically attach, format, and mount a volume for a deployed pod. This approach requires an existing ScaleIO volume.
+
+### Create Volume
+
+Static persistent volumes require that the volume, to be consumed by the pod, be already created in ScaleIO.  You can use your ScaleIO tooling to create a new volume or use the name of a volume that already exists in ScaleIO.  For this demo, we assume there's a volume named `vol-0`.  If you want to use an existing volume, ensure its name is reflected properly in the `volumeName` attribute below.
+
+### Deploy Pod YAML
+
+Create a pod YAML file that declares the volume (above) to be used.
+
+File: [pod.yaml](pod.yaml)
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-0
+spec:
+  containers:
+  - image: gcr.io/google_containers/test-webserver
+    name: pod-0
+    volumeMounts:
+    - mountPath: /test-pd
+      name: vol-0
+  volumes:
+  - name: vol-0
+    scaleIO:
+      gateway: https://localhost:443/api
+      system: scaleio
+      volumeName: vol-0
+      secretRef:
+        name: sio-secret
+      fsType: xfs
+```
+
+Notice the followings in the previous YAML:
+
+- Update the `gatewway` to point to your ScaleIO gateway endpoint.  
+- The `volumeName` attribute refers to the name of an existing volume in ScaleIO.  
+- The  `secretRef` attribute references the name of the secret object deployed earlier.
+
+Next, deploy the pod.
+
+```
+$> kubectl create -f examples/volumes/scaleio/pod.yaml
+```
+You can verify the pod:
+```
+$> kubectl get pod
+NAME      READY     STATUS    RESTARTS   AGE
+pod-0     1/1       Running   0          33s
+```
+Or for more detail, use 
+```
+kubectl describe pod pod-0
+```
+You can see the attached/mapped volume on the node:
+```
+$> lsblk
+NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+...
+scinia      252:0    0    8G  0 disk /var/lib/kubelet/pods/135986c7-dcb7-11e6-9fbf-080027c990a7/volumes/kubernetes.io~scaleio/vol-0
+```
+
+## StorageClass and Dynamic Provisioning
+
+In this example, we will see how the ScaleIO volume plugin can automatically provision a new volume as described in a `StorageClass`.
+
+### StorageClass
+
+Define a new `StorageClass` as shown in the following YAML.
+
+File [sc.yaml](sc.yaml)
+
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: sio-small
+provisioner: kubernetes.io/scaleio
+parameters:
+  gateway: https://localhost:443/api
+  system: scaleio
+  protectionDomain: default
+  secretRef: sio-secret
+  fsType: xfs
+```
+
+Note the followings:
+
+- The `name` attribute is set to `sio-small` . It will be referenced later.
+- The `provisioner` attribute is set to `kubernetes.io/scaleio` to trigger the ScaleIO plugin.
+- The use of the `parameters:` section in the yaml for configurations.  
```

Comment:
@rootfs There are several lines shown above, which are  you referring to, line `175` ?